From 5fbf1f118233507bfdc4510b7f90de1bb8ffd1c4 Mon Sep 17 00:00:00 2001
From: Stephane Danieau <stephane.danieau@st.com>
Date: Mon, 9 Sep 2024 10:47:39 +0200
Subject: [PATCH] arm64: defconfig: add USB_CONFIGFS_F_UVC in fragment-02

It provides a userspace API to process UVC control requests
and stream video data to the host.

Change-Id: I6762de31561b197e6d0d0cbfd9cb7d037a0e1f54
Signed-off-by: Stephane Danieau <stephane.danieau@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/403008
Domain-Review: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Stephane DANIEAU <stephane.danieau@foss.st.com>
Reviewed-by: Stephane DANIEAU <stephane.danieau@foss.st.com>
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -111,6 +111,7 @@ CONFIG_TOUCHSCREEN_ILI210X=y
 CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
 CONFIG_UCSI_STM32G0=m
+CONFIG_USB_CONFIGFS_F_UVC=y
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ONBOARD_HUB=y
